I've got breakage of sentry sdk on darwin on master with the following 3 failures
> FAILED tests/profiler/test_continuous_profiler.py::test_continuous_profiler_auto_start_and_stop_sampled[non-experiment-thread]
       > FAILED tests/profiler/test_continuous_profiler.py::test_continuous_profiler_auto_start_and_stop_sampled[experiment-thread]
       > FAILED tests/test_basics.py::test_stacktrace_big_recursion

just saying that if you were thinking of mering to master disabling flaky tests, that is also possilbe.

--- a/pkgs/development/python-modules/sentry-sdk/default.nix
+++ b/pkgs/development/python-modules/sentry-sdk/default.nix
@@ -198,7 +198,9 @@ buildPythonPackage rec {
     "test_auto_session_tracking_with_aggregates"
     # timing sensitive
     "test_profile_captured"
-    "test_continuous_profiler_manual_start_and_stop"
+    "test_continuous_profiler_auto"
+    "test_continuous_profiler_manual"
+    "test_stacktrace_big_recursion"
     # assert ('socks' in "<class 'httpcore.connectionpool'>") == True
     "test_socks_proxy"
     # requires socksio to mock, but that crashes pytest-forked

